### PR TITLE
Incorrect field access modifier after action generate set/get methods. Can't use action generate set/get methods for static fields. (Issue #442)

### PR DESCRIPTION
**Pull request by bynuff**
#442

### CODE REVIEW

**Comment by eliasku** [comment]
OK. I've reviewed your changes, all looks good. Please, next time add summary description, something like this:

Fix issue #442.
- Making getter/setter generation available for static fields
- Fixing making `var` property declaration public automatically
- Release notes updated
- Test cases added for static variables

**Comment by bynuff** [comment]
@eliasku OK, thx.

File: src/META-INF/plugin.xml [hunk 1]
```diff
@@ -69,6 +69,7 @@
         <li>Save and restore OpenFL arguments on the Haxe Module settings dialog. (TiVo Issue #74)</li>
         <li>Fixed repainting issue for OpenFL dialog when selecting compiler types. (TiVo Issue #44)</li>
         <li>Fixed error parsing for Windows platforms.</li>
+        <li>Incorrect field access modifier after action generate set/get methods. Can't use action generate set/get methods for static fields. (TiVo Issue #442)</li>
```
Comment (EBatTiVo) [inline src/META-INF/plugin.xml]: Wrong position.  It should not have be placed in an old release section.  It should have been inserted into the "unreleased changes" section before line 49.

Comment (bynuff) [inline src/META-INF/plugin.xml]: corrected

**Pull request merged**